### DISCUSSION TITLE
Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by david-c-kline** [microsoft/MixedRealityToolkit-Unity#9027] [General]
Please feel free to use the new discussions area to ask a question, share ideas or showcase something cool you have built using MRTK.

Thanks!
MRTK Team

**Comment by ghost** [comment]
Hi david,
this is a great idea, thank you.
Take a look at the posts again. Maybe you can help me with my question about the HP reverb g2.
Thanks
SaschaFFm
